File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.js
const [a,b,c,d,e,f] = require("fs").readFileSync("/dev/stdin").toString().trim().split(" ").map(Number);

let [x,y] = [0,0];

for(let i=-999; i<=999; i++){
    for(let j=-999; j<=999; j++){
        if((a*i)+(b*j) === c && (d*i)+(e*j) === f){
         x += i;
         y += j;   
        }
    }
}

console.log(`${x} ${y}`)

File: 백준/Bronze/19532. 수학은 비대면강의입니다/수학은 비대면강의입니다.py
import sys
a,b,c,d,e,f = map(int, sys.stdin.readline().split())

for x in range(-999,1000):
    for y in range(-999,1000):
        if (a*x)+(b*y) == c and (d*x)+(e*y) == f:
            print(x,y)
            break